### PR TITLE
Refactoring 1tb longevity

### PR DESCRIPTION
**Pull request by roydahan**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by roydahan** [comment]
Waiting first for testing https://github.com/scylladb/scylla-cluster-tests/pull/3556.
Then, we will switch to this longevity.

**Comment by roydahan** [comment]
After merging  need to rename the jenkins job from 7 days to 5 days.

**Comment by roydahan** [comment]
> Waiting first for testing #3556.
> Then, we will switch to this longevity.

Tests passed.
Switching officially to this.

**Comment by roydahan** [comment]
@amoskong / @fgelcer / @bentsi please review, I need this on 4.5 very soon.

**Review by fgelcer — CHANGES_REQUESTED** [review]
all the 3 pipelines must also be renamed to `*longevity-1tb-5days*.jenkinsfile`

File: jenkins-pipelines/ics-longevity-1tb-7days.jenkinsfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ longevityPipeline(
     backend: 'aws',
     aws_region: 'eu-west-1',
     test_name: 'longevity_test.LongevityTest.test_custom_time',
-    test_config: '''["test-cases/longevity/longevity-1TB-7days-authorization-and-tls-ssl.yaml", "test-cases/enterprise-features/ics/longevity/ics-longevity-1TB-7days-authorization-and-tls-ssl.yaml"]''',
+    test_config: '''["test-cases/longevity/longevity-1TB-5days-authorization-and-tls-ssl.yaml", "test-cases/enterprise-features/ics/longevity/ics-longevity-1TB-7days-authorization-and-tls-ssl.yaml"]''',
```
Comment (amoskong) [inline jenkins-pipelines/ics-longevity-1tb-7days.jenkinsfile:10]: "test-cases/enterprise-features/ics/longevity/ics-longevity-1TB-7days-authorization-and-tls-ssl.yaml" is used, and the cs duration is still 7days (duration=10080m).

We might need to change it together?

Comment (roydahan) [reply jenkins-pipelines/ics-longevity-1tb-7days.jenkinsfile:10]: Not critical to change (just renaming) and don't want to test the jobs .

**Pull request merged**